lib/coinbase.js

```diff
@@ -22,16 +22,17 @@ export default class Coinbase {
     return Promise.all(
       this.cryptoCurrencies.map(cryptoCurrency =>
         this.getPrice({ action, currencyPair: `${cryptoCurrency}-${this.localCurrency}` })
-          .then((price) => {
+          .then((value) => {
             const body = {
               amount: 1,
               currency: cryptoCurrency,
               localCurrency: this.localCurrency,
-              price,
+              value,
               metricName: `${action}Price`,
+              namespace: `Coinbase`,
```

Comment:
Please leave as is... All metrics should be within the main `coinbase` namespace. If you want to make them more specific please update `cloudWatch.js` to include something like:

```js
static setupBuySellDimensions(currency, localCurrency, provider = 'coinbase') {
    return [
      {
        Name: 'Provider',
        Value: provider,
      },
      {
        Name: 'CryptoCurrency',
        Value: currency,
      },
      {
        Name: 'LocalCurrency',
        Value: localCurrency,
      },
      {
        Name: 'Stage',
        Value: process.env.STAGE || 'dev',
      },
    ];
  }
```